model.py

```diff
@@ -2738,27 +2743,29 @@ class Edition(Base):
         PERIODICAL_MEDIUM : u"http://schema.org/PublicationIssue",
         MUSIC_MEDIUM :  u"http://schema.org/MusicRecording",
         VIDEO_MEDIUM :  u"http://schema.org/VideoObject",
+        IMAGE_MEDIUM :  u"http://schema.org/ImageObject",
+        COURSEWARE_MEDIUM : u""
     }
 
     additional_type_to_medium = {}
     for k, v in medium_to_additional_type.items():
         additional_type_to_medium[v] = k
 
     medium = Column(
-        Enum(BOOK_MEDIUM, PERIODICAL_MEDIUM, AUDIO_MEDIUM,
-             MUSIC_MEDIUM, VIDEO_MEDIUM, name="medium"),
+        Enum(BOOK_MEDIUM, PERIODICAL_MEDIUM, AUDIO_MEDIUM, MUSIC_MEDIUM, VIDEO_MEDIUM, IMAGE_MEDIUM, COURSEWARE_MEDIUM,
+             name="medium"),
```

Comment:
This is going to need a migration script. See `core/migration/20170928-add-registered-coverage-status.sql` for an example. It would look something like this:

```
ALTER TYPE medium ADD VALUE IF NOT EXISTS 'image' AFTER 'video';
ALTER TYPE medium ADD VALUE IF NOT EXISTS 'courseware' AFTER 'image';
```

Reply:
Ok, created and added !!